packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -235,7 +235,7 @@ export default function MainFeedLayout({
   const header = (
     <LayoutHeader className="flex-row">
       {!isSearchOn && getFeedTitle()}
-      <div className="flex flex-row flex-wrap gap-4 items-center mr-px">
+      <div className="flex flex-row flex-wrap gap-4 items-center py-2 mr-px">
```

Comment:
Looks like an overflow issue then?
Perhaps we can rather leverage border instead of the box-shadow for the border then?